### PR TITLE
In Progress: Add gangstr

### PR DESCRIPTION
**Pull request by christopher-schroeder**
Add GangSTR

### CODE REVIEW

**Comment by christopher-schroeder** [comment]
@bioconda-bot please fetch artifacts

**Comment by christopher-schroeder** [comment]
@BiocondaBot please update

File: recipes/gangstr/build.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX="${PREFIX}" ..
+make install
+# TODO: copy python scripts
```
Comment (mbargull) [inline recipes/gangstr/build.sh:6]: Those are some Python 2 scripts... Are they important? Do we need/want to include them?

Comment (christopher-schroeder) [reply recipes/gangstr/build.sh:6]: no, this is a relict, we do not need them

Comment (mbargull) [reply recipes/gangstr/build.sh:6]: Perfect :)

**Review by mbargull — APPROVED** [review]

**Comment by mbargull** [comment]
@BiocondaBot, please fetch artifacts

**Comment by mbargull** [comment]
@BiocondaBot, please merge

**Pull request merged**